src/components/Map.js

```diff
@@ -48,7 +44,6 @@ class Map extends Component {
         ref="map"
         className="venues__map"
         center={this.props.startPosition}
-        onLeafletPopupopen={this.handleClick}
```

Comment:
That was my previous attempt just before I went away. This is the new method:
https://github.com/shortlist-digital/agreable-venues-plugin/pull/4/files#diff-76b9295b285377ed8b719b2f197ab99bR61
Essentially not using react for the marker click event (Leaflet instead) but immediately passing responsibility off to the routing.